src/routes/event-match.tsx

```diff
@@ -61,19 +63,22 @@ const showEventResults = 'Event Results'
 
 type SelectedDisplay = typeof showMatchResults | typeof showEventResults
 
+// eslint-disable-next-line complexity
```

Comment:
I don't think this eslint rule is triggering on the next line, so I think we can take this out